kubeflow/asm/deprecated/README.md

```diff
@@ -0,0 +1,47 @@
+# Anthos Service Mesh (ASM)
```

Comment:
We can add a note here saying this instruction uses `install_asm` and is only feasible for pre-ASM 1.10 versions.